assets/lokomotive-kubernetes/bootkube/resources/charts/lokomotive/values.yaml

```diff
@@ -0,0 +1,40 @@
+certificates:
```

Comment:
Except the certificates (which are actual variables), I'd inline all the values.